tests/test_flow_order_guardrail.py

```
@@ -383,8 +383,9 @@ def test_pattern_detects_multiline_list(self):
         # This won't be detected by current patterns since they're single-line
         # That's acceptable - the main risk is copy-paste of inline lists
         # Multiline lists are less common and easier to spot in review
-        violations = _find_violations(code, Path("test.py"))
+        _violations = _find_violations(code, Path("test.py"))  # noqa: F841
```

Ruff (0.14.10)

Unused `noqa` directive (unused: `F841`)

Remove unused `noqa` directive

(RUF100)